package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "point-of-view",
-  "version": "4.0.0",
+  "version": "4.1.0",
```

Can you please revert this?

```suggestion
  "version": "4.0.0",
```